api/src/test/java/org/openmrs/api/CohortServiceTest.java

```diff
@@ -440,9 +444,9 @@ public void getCohortsContainingPatient_shouldNotReturnVoidedCohorts() throws Ex
 	public void getCohortsContainingPatient_shouldReturnCohortsThatHaveGivenPatient() throws Exception {
```

Comment:
I added comments somewhere else, the membership might be getting saved but not all the fields on it, I believe specifically the cohort field isn't getting saved because you are not setting it in the first place